### PR TITLE
added two pieces of information which helped me setting up my environment

### PR DESCRIPTION
**Pull request by FuchsDominik**
No changes in testable code.

### Motivation and Context

I personally had some problems setting up the local environment and this information would have helped me.

### Description

Added two lines of text in the md file SETUP.md

### CODE REVIEW

File: doc/setup/SETUP.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Before you can build Artemis, you must install and configure the following depen
...
 3. [Node.js](https://nodejs.org): We use Node (>=12.8.0) to compile and run the client Angular application. Depending on your system, you can install Node either from source or as a pre-packaged bundle.
 4. [Yarn](https://yarnpkg.com): We use Yarn (>=1.17.3) to manage client side Node dependencies.
-Depending on your system, you can install Yarn either from source or as a pre-packaged bundle.
+Depending on your system, you can install Yarn either from source or as a pre-packaged bundle. The **global install** of step 2, Installation, of the "Getting started" tab is sufficient to run the application.
```
Comment (krusche) [inline doc/setup/SETUP.md:14]: I don't understand this comment. What is step 2? What is the "Getting started" tab?
I think we could just reference https://classic.yarnpkg.com/en/docs/install here

Comment (FuchsDominik) [reply doc/setup/SETUP.md:14]: It's indeed easier to just provide a link than describing it textual. Thanks!

Comment (krusche) [reply doc/setup/SETUP.md]: Can you change this then please and merge the changes from develop?

File: doc/setup/SETUP.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ info:
     imprint: https://ase.in.tum.de/lehrstuhl_1/component/content/article/179-imprint
 ```
 Change all entries with ```<...>``` with proper values, e.g. your TUM Online account credentials to connect to the given instances of JIRA, Bitbucket and Bamboo. Alternatively, you can connect to your local JIRA, Bitbucket and Bamboo instances.
-Be careful that you don't commit changes in this file. Best practice is to specify that your local git repository ignores this file or assumes that this file is unchanged.
+Be careful that you don't commit changes in this file. Best practice is to specify that your local git repository ignores this file or assumes that this file is unchanged. Also note that except for the credentials and a random encryption-password, other fields like `ldap` can be left blank.
```
Comment (jpbernius) [inline doc/setup/SETUP.md:95]: You do not even need the credentials and the default encryption-password also works. 😋

Comment (krusche) [reply doc/setup/SETUP.md:95]: you might want to reference https://github.com/ls1intum/Artemis/blob/develop/doc/jenkins-gitlab/JENKINS_AND_GITLAB_SETUP.md and https://github.com/ls1intum/Artemis/blob/develop/doc/bamboo-bitbucket-jira/BAMBOO_BITBUCKET_JIRA_SETUP.md for more details about the yml config

Comment (FuchsDominik) [reply doc/setup/SETUP.md]: Done

File: doc/setup/SETUP.md (hunk 1 above)
Comment (FuchsDominik) [reply doc/setup/SETUP.md]: Done

File: doc/setup/SETUP.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ Before you can build Artemis, you must install and configure the following depen
...
-3. [Node.js](https://nodejs.org): We use Node (>=13.12.0) to compile and run the client Angular application. Depending on your system, you can install Node either from source or as a pre-packaged bundle.
-4. [Yarn](https://classic.yarnpkg.com): We use Yarn 1.x (>=1.22.0) to manage client side Node dependencies.
-Depending on your system, you can install Yarn either from source or as a pre-packaged bundle.
+3. [Node.js](https://nodejs.org): We use Node (>=12.8.0) to compile and run the client Angular application. Depending on your system, you can install Node either from source or as a pre-packaged bundle.
```
Comment (krusche) [inline doc/setup/SETUP.md:12]: please revert the change in this line.

File: doc/setup/SETUP.md [hunk 4]
```diff
@@ -9,9 +9,9 @@ Before you can build Artemis, you must install and configure the following depen
...
+4. [Yarn](https://yarnpkg.com): We use Yarn (>=1.17.3) to manage client side Node dependencies.
```
Comment (krusche) [inline doc/setup/SETUP.md:13]: please revert the change in this line.

Comment (FuchsDominik) [reply doc/setup/SETUP.md]: Thanks, done.
